<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGames</class>
 <widget class="QWidget" name="FormGames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>493</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ebrima</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SteamAchievementsStatistic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="GroupBoxFilter">
     <property name="title">
      <string>     Фильтр</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="LineEditGame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Найти игру</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonFind">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string> Найти</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="MultiSelectComboBox" name="ComboBoxGroups"/>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckBoxFavorites">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Избранные</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="TableGames"/>
   </item>
   <item>
    <widget class="QProgressBar" name="ProgressBarLoading">
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string notr="true">%v/%m</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiSelectComboBox</class>
   <extends>QComboBox</extends>
   <header>class\NotMine\MultiSelectComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ButtonFind</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
